frontend/public/components/app.jsx

```diff
@@ -227,7 +227,10 @@ if ('serviceWorker' in navigator) {
 render(
   <Provider store={store}>
     <Router history={history} basename={window.SERVER_FLAGS.basePath}>
-      <Route path="/" component={App} />
+      <Switch>
+        <Route path="/cloudshell" component={CloudShellTab} />
```

Comment:
UX doesn't want terminology of cloud shell used anywhere. Use terminal instead:
```suggestion
        <Route path="/terminal" component={CloudShellTab} />
```